sherpa/astro/tests/test_astro_data2.py

```diff
@@ -85,6 +85,68 @@ def test_need_numpy_channels():
     assert pha.get_filter() == 'No noticed bins'
 
 
+@pytest.mark.parametrize("chtype,expected,args",
+                         [("channel", '1:10', []),
+                          ("channel", '', [(False, 1, 10)]),
+                          ("channel", '2:9', [(True, 2, 9)]),
+                          ("channel", '2:3,7:9', [(True, 2, 9), (False, 4, 6)]),
```

Comment:
All these tests are doing is to test the current behavior (since I don't think we have a principled design document to explain what they do now). You'd have to jump ahead to #1219 to check what it is changed to ....

Reply:
Understood. I was not asking for a change, just for clarification in which order things are happening.